core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/operators/OperatorDescriptor.scala

```diff
@@ -122,7 +123,8 @@ import edu.uci.ics.texera.workflow.operators.udf.pythonV1.PythonUDFOpDesc
     new Type(value = classOf[CSVOldScanSourceOpDesc], name = "CSVOldFileScan"),
     new Type(value = classOf[RedditSearchSourceOpDesc], name = "RedditSearch"),
     new Type(value = classOf[PythonLambdaFunctionOpDesc], name = "PythonLambdaFunction"),
-    new Type(value = classOf[DownloadOpDesc], name = "Download")
+    new Type(value = classOf[BulkDownloadOpDesc], name = "BulkDownload"),
```

Comment:
call it `BulkDownloader` to be consistent with `URLFetcher`

Reply:
done